### DISCUSSION TITLE
Edit selected node in VueFlow

### DISCUSSION THREAD

**Discussion by tzelovaka** [bcakmakoglu/vue-flow#1078] [Q&A]
Hello everyone! I want to edit text in selected node of my scheme created with VueFlow, but I have no idea to do this.

**Comment by bcakmakoglu** [comment]
Mh... I'm not sure I follow you.
What are you exactly trying to do? Use a node with an input field?
Custom Nodes can literally be anything, so you can just create a custom node with an input field and that's it, there's no magic to it 😄

**Reply by tzelovaka** [reply]
Can you show any example pls (node with an input field)? + can you write how i can get through "getSelectedNode" id of selected node. I sorry, I'm beginner and started to learn that library only yesterday((
